mesonbuild/compilers/rust.py

```diff
@@ -196,6 +196,20 @@ def get_debug_args(self, is_debug: bool) -> T.List[str]:
     def get_optimization_args(self, optimization_level: str) -> T.List[str]:
         return rust_optimization_args[optimization_level]
 
+    def build_rpath_args(self, env: 'Environment', build_dir: str, from_dir: str,
+                         rpath_paths: T.Tuple[str, ...], build_rpath: str,
+                         install_rpath: str) -> T.Tuple[T.List[str], T.Set[bytes]]:
+        rpath_args, rpath_dirs_to_remove = super().build_rpath_args(
+            env, build_dir, from_dir, rpath_paths, build_rpath, install_rpath)
```

Comment:
Since these variables are totally internal to this function, I'd call it:

```suggestion
        args, to_remove = super().build_rpath_args(env, build_dir, from_dir, rpath_paths, 
                                                   build_rpath, install_rpath)
```

which makes it a lot easier to read (and follows the internal style)